build-logic/src/main/kotlin/io/github/droidkaigi/confsched2023/primitive/AndroidFirebasePlugin.kt

```diff
@@ -22,6 +22,7 @@ class AndroidFirebasePlugin : Plugin<Project> {
                 }
             }
             dependencies {
+                implementationPlatform(libs.library("firebaseBom"))
                 implementation(libs.library("firebaseAuth"))
                 implementation(libs.library("firebaseCommon"))
                 implementation(libs.library("multiplatformFirebaseAuth"))
```

Comment:
It seems that this dependency (`"multiplatformFirebaseAuth"`) can be removed from here because this library is not used in the module that applies this plugin (i.e. `:app-android`) 👀

Reply:
We can remove it but this pr's implementation is better than current implementation!